synoptic/src/synoptic/model/export/JsonExporter.java

```diff
@@ -66,22 +51,51 @@ public EventInstance(int traceID, int eventIndexWithinTrace) {
         assert graph instanceof PartitionGraph;
         PartitionGraph pGraph = (PartitionGraph) graph;
 
-        // EvBasedGraph evGraph = new EvBasedGraph(pGraph);
-        // System.out.println("evGraph:\n" + evGraph);
+        EvBasedGraph evGraph = new EvBasedGraph(pGraph);
+        System.out.println("evGraph:\n" + evGraph);
 
         Map<String, Object> finalModelMap = new LinkedHashMap<String, Object>();
 
-        // Add log to final model map
-        List<Map<String, Object>> logListOfTraces = makeLogJSON(pGraph);
-        finalModelMap.put("log", logListOfTraces);
+        // Add displayables to final model map
+        List<Map<String, Object>> displayablesList = makeDisplayablesJSON(
+                evGraph);
+
+        // Add nodes to the final model map
+        List<Map<String, Integer>> nodesList = makeNodesJSON(evGraph);
+        finalModelMap.put("nodes", nodesList);
 
-        // Add partitions to final model map
-        List<Map<String, Object>> partitionList = makePartitionsJSON(pGraph);
-        finalModelMap.put("partitions", partitionList);
+        // Add edges to final model map
+        List<Map<String, Integer>> edgesList = makeEdgesJSON(evGraph);
+        finalModelMap.put("edges", edgesList);
+
+        finalModelMap.put("displayables", displayablesList);
+
+        // Add event types to final model map
+        List<Map<String, Object>> eventTypesList = makeEventTypesJSON(evGraph);
+        finalModelMap.put("eventTypes", eventTypesList);
+
+        // Add events to final model map
+        List<Map<String, Object>> eventsList = makeEventsJSON(evGraph);
+        finalModelMap.put("events", eventsList);
+
+        // Add invariant types to final model map
+        List<Map<String, Object>> invariantTypesList = makeInvariantTypesJSON(
+                pGraph);
+        finalModelMap.put("invariantTypes", invariantTypesList);
 
         // Add invariants to final model map
-        List<Map<String, Object>> invariantList = makeInvariantsJSON(pGraph);
-        finalModelMap.put("invariants", invariantList);
+        List<Map<String, Object>> invariantsList = makeNewInvariantsJSON(
```

Comment:
Why does the method name have `New` in its name? Unless this means something, remove it.